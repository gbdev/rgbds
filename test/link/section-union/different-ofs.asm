IF !DEF(SECOND)
	def ATTRS equs ",ALIGN[3,7]"
ELSE
	def ATTRS equs ",ALIGN[3,6]"
ENDC

SECTION UNION "conflicting alignment", WRAM0 ATTRS
	db

	PURGE ATTRS
