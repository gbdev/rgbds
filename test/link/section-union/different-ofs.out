error: Section "conflicting alignment" is defined with conflicting 8-byte alignment (offset 7) and 8-byte alignment (offset 6)
---
error: <stdin>(18):
    Section already declared with incompatible 8-byte alignment (offset 7)
FATAL: <stdin>(18):
    Cannot create section "conflicting alignment" (1 error)
